data/json/furniture_and_terrain/furniture-plumbing.json

```diff
@@ -55,6 +57,7 @@
     "type": "furniture",
     "id": "f_sink",
     "name": "sink",
+    "looks_like": "t_machinery_light",
```

Comment:
This one again is kind of a base-level furniture, and could loop around to maybe f_bathtub